SS14.Admin/Pages/Settings/Index.cshtml

```diff
@@ -8,10 +8,43 @@
     var textClass = darkMode ? "text-light" : "text-dark";
 }
 
-@* TODO: Make this actually look good*@
-<div class="container">
-    <h1>Settings</h1>
-    <button type="button" class="btn btn-primary" onclick="toggleDarkMode()">Toggle darkmode</button>
+<style>
+    .description {
+        transition: height 0.5s ease-in-out;
+        overflow: hidden;
+        height: 0;
+        border-radius: 5px; /* Rounded corners */
+    }
+</style>
+
+<div class="container py-4">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <h1 class="mb-4">Settings</h1>
+            <button type="button" class="btn btn-primary mb-4" onclick="toggleDarkMode()">Toggle darkmode</button>
+        </div>
+    </div>
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <h1 class="mb-4">Changelog</h1>
+            <ul class="list-group border">
+                @foreach(var change in Model.Changes.OrderByDescending(c => c.Id))
```

Comment:
This would be fixed by the schema changed I posted above, but I just want to point out here:

Because you're using strings as `Id`, your ordering breaks the moment you get to ID 10:

![image](https://github.com/space-wizards/SS14.Admin/assets/8107459/5e6b9e5e-581f-4384-947f-6172bd9f23b8)

Reply:
this is dumb, and i forgot this is the hacky way I did it to make it work, rewriting

Reply:
This has been changed :)